Header ArduinoCommand
float64 link1AngleDeg
float64 link2AngleDeg
float64 link3AngleDeg